k8s/redis/deployment.yaml

```
@@ -0,0 +1,67 @@
+  template:
+    metadata:
+      labels:
+        app: redis
+    spec:
+      containers:
+      - name: redis
+        image: redis:7.2-alpine
+        ports:
+        - containerPort: 6379
+        command:
+          - redis-server
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)